src/components/v1/V1Project/modals/V1ConfirmPayOwnerModal/index.tsx

```diff
@@ -96,7 +98,11 @@ export default function V1ConfirmPayOwnerModal({
       title={t`Pay ${metadata.name}`}
       visible={visible}
       onOk={pay}
-      okText={t`Pay`}
+      okText={
+        userAddress
+          ? payButtonText
+          : t`Connect wallet to ${payButtonText?.toLowerCase()}`
```

Comment:
I think we should keep this as `Connect wallet to pay` - there's so much variation in the pay button text, sometimes it wont make sense.

Reply:
Yeah good point. Will revert